src/SFA.DAS.Forecasting.Domain/Services/IPaymentForecastService.cs

```diff
@@ -0,0 +1,9 @@
+﻿using System.Threading.Tasks;
+
+namespace SFA.DAS.Forecasting.Domain.Services
+{
+    public interface IPaymentForecastService
+    {
+        Task TriggerPaymentForecast(short periodMonth, int periodYear, string periodEnd, long accountId);
```

Comment:
Can the parameters be specified in single object? e.g. ForecastSettings or ForecastRequest.

Reply:
Naming as above.

Reply:
if there were more than 3 or 4 then I would make this change, or if it was likely to change i.e more properties added.